File: backend/src/main/java/com/example/shoestore/dto/response/CartItemResponse.java
package com.example.shoestore.dto.response;

import lombok.*;
import java.math.BigDecimal;

@Getter @Setter @AllArgsConstructor @NoArgsConstructor
public class CartItemResponse {
    private Integer cartItemId;
    private Integer productId;
    private Integer variantId;
    private String productName;
    private String imageUrl;
    private String color;
    private String size;
    private BigDecimal price;
    private Integer quantity;
    private Integer stockQuantity;
}

File: android/app/src/main/java/com/example/shoeshop/dto/respone/CartItemResponse.kt
package com.example.shoeshop.dto.respone

import java.io.Serializable

data class CartItemResponse(
    val cartItemId: Int,
    val productId: Int,
    val variantId: Int,
    val productName: String,
    val imageUrl: String,
    val color: String,
    val size: String,
    val price: Double,
    var quantity: Int,
    val stockQuantity: Int,
    var isChecked: Boolean = false
) : Serializable